GOES/Scripts/update_goes_differential_page.py

```diff
@@ -513,10 +565,65 @@ def adjust_format(val):
     
     return out
 
+def send_mail(content, subject, admin):
+    """
+    send out a notification email to admin
```

Comment:
A more comprehensive description of the notification sent would be helpful because the code contains multiple send_email functions.